pom.xml

```diff
@@ -376,7 +375,7 @@
                         <exclude>**/src/test/resources/**</exclude>
                         <exclude>**/target/**</exclude>
                     </excludes>
-                    <lineEnding>LF</lineEnding>
+                    <lineEnding>AUTO</lineEnding>
```

Comment:
this ain't a good idea

AUTO for windows is different from mac that is in time different from linux.

So our build become non-deterministic